contracts/utils/Operators.sol

```diff
@@ -77,13 +74,12 @@ contract Operators is Ownable, IOperators {
     /**
      * @dev Allows an operator to execute arbitrary functions on any smart contract.
      * Can only be called by an operator.
-     * @param target address of the contract to execute the function on
+     * @param target address of the contract to execute the function on. Existence is not checked.
      * @param callData ABI encoded function call to execute on target
      * @return data return data from executed function call
      */
-    function execute(address target, bytes calldata callData) external onlyOperator returns (bytes memory) {
-        // solhint-disable-next-line avoid-low-level-calls
-        (bool success, bytes memory data) = target.call(callData);
+    function execute(address target, bytes calldata callData) external payable onlyOperator returns (bytes memory) {
```

Comment:
based on a change we're adding to the express service, there's a need to allow configuring the msg value being forwarded
```suggestion
    function execute(address target, bytes calldata callData, unit256 nativeValue) external payable onlyOperator returns (bytes memory) {
```

Reply:
Changed this in the interface as well